src/test_typing_extensions.py

```diff
@@ -4320,6 +4320,24 @@ class Y(Generic[T], NamedTuple):
         self.assertEqual(Y.__orig_bases__, (Generic[T], NamedTuple))
         self.assertEqual(Y.__mro__, (Y, Generic, tuple, object))
 
+        class Z(Y[int]):
+            ...
+        self.assertEqual(Z.__bases__, (Y,))
+        self.assertEqual(Z.__orig_bases__, (Y[int],))
+        self.assertEqual(Z.__mro__, (Z, Y, Generic, tuple, object))
+
+        @runtime_checkable
+        class Proto(Protocol):
+            @property
+            def x(self):
+                ...
+
+        z = Z(3)
+        self.assertIs(type(z), Z)
+        self.assertIsInstance(z, Z)
+        self.assertIsInstance(z, Proto)
+        self.assertEqual(z.x, 3)
```

Comment:
This feels like an odd place to put a test for this issue, as this class is dedicated for `NamedTuple` tests, and this issue has nothing to do with `NamedTuple`s.

I would create a new method in the `ProtocolTests` class here, and add your test in the new method:

https://github.com/python/typing_extensions/blob/f62fa3f37ce1949e8a536691b30c59c4889445eb/src/test_typing_extensions.py#L1678